Core/Src/obc_handler.c

```diff
@@ -79,10 +81,13 @@ int obc_handle_command(uint8_t obc_cmd) {
         uint8_t buffer[sizeof(hk)];
         memcpy(buffer, &hk, sizeof(hk));
         obc_spi_transmit(buffer, sizeof(buffer));
+        iris_log("Done sending housekeeping data");
         return 0;
     }
     case IRIS_TAKE_PIC: {
         take_image();
+        DBG_PUT("Image capture complete");
```

Comment:
iris_log should send the logs to DBG_PUT if configured to do so, instead of having two calls with the same string